src/rgw/rgw_cr_rados.cc

```diff
@@ -73,9 +73,9 @@ RGWAsyncRadosProcessor::RGWAsyncRadosProcessor(CephContext *_cct, int num_thread
   : cct(_cct), m_tp(cct, "RGWAsyncRadosProcessor::m_tp", "rados_async", num_threads),
     req_throttle(_cct, "rgw_async_rados_ops", num_threads * 2),
     req_wq(this,
-	   ceph::make_timespan(g_conf()->rgw_op_thread_timeout),
-	   ceph::make_timespan(g_conf()->rgw_op_thread_suicide_timeout),
-	   &m_tp) {
+       ceph::make_timespan(g_conf()->rgw_op_thread_timeout),
```

Comment:
nit: please avoid indentation changes in the file. here as well as lines 77, 78, 112, 130, 667